worlds/pokemon_emerald/rules.py

```diff
@@ -1370,17 +1371,17 @@ def get_location(location: str):
     # Route 129
     set_rule(
         get_entrance("REGION_UNDERWATER_ROUTE129/MARINE_CAVE_ENTRANCE_1 -> REGION_UNDERWATER_MARINE_CAVE/MAIN"),
-        lambda state: hm_rules["HM08 Dive"](state) and \
-            state.has("EVENT_DEFEAT_CHAMPION", world.player) and \
-            state.has("MARINE_CAVE_ROUTE_129_1", world.player) and \
-            state.has("EVENT_DEFEAT_SHELLY", world.player)
+        lambda state: hm_rules["HM08 Dive"](state)
+            and state.has("EVENT_DEFEAT_CHAMPION", world.player)
+            and state.has("MARINE_CAVE_ROUTE_129_1", world.player)
+            and state.has("EVENT_DEFEAT_SHELLY", world.player)
     )
     set_rule(
         get_entrance("REGION_UNDERWATER_ROUTE129/MARINE_CAVE_ENTRANCE_2 -> REGION_UNDERWATER_MARINE_CAVE/MAIN"),
-        lambda state: hm_rules["HM08 Dive"](state) and \
-            state.has("EVENT_DEFEAT_CHAMPION", world.player) and \
-            state.has("MARINE_CAVE_ROUTE_129_2", world.player) and \
-            state.has("EVENT_DEFEAT_SHELLY", world.player)
+        lambda state: hm_rules["HM08 Dive"](state)
+            and state.has("EVENT_DEFEAT_CHAMPION", world.player)
+            and state.has("MARINE_CAVE_ROUTE_129_2", world.player)
+            and state.has("EVENT_DEFEAT_SHELLY", world.player)
```

Comment:
you can do this?? i swore you had to use the `\` line breaks here

Reply:
Oh wow, I didn't think this was possible either. I usually put parens around the full thing. So like `lambda state: (thing and thing2 spanning multiple lines)